test/app-media-audio.html

```diff
@@ -17,9 +17,11 @@
 
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
     <script src="../../webrtc-adapter/release/adapter.js"></script>
+    <!-- npm version has different dir structure compared to bower -->
```

Comment:
Maybe we should do something hacky to fix bower. It seems bad and potentially misleading to attempt to load a script that doesn't exist in all cases.